### PR TITLE
Adding Cumulus Image and Proof of Concept Lesson

### PR DESCRIPTION
**Pull request by cloudtoad**
Add Cumulus image and demo topology to curriculum.  PTR only.  Uses the python configuration option (specified in the lesson.meta.yaml file) to copy configuration files to the booted images, then restarts associated processes.

### CODE REVIEW

**Comment by Mierdin** [comment]
Is this ready to go from your perspective? I should have time later in the week to check this out and merge but if you have more changes I'll wait to take a look.

**Comment by cloudtoad** [comment]
> Is this ready to go from your perspective? I should have time later in the week to check this out and merge but if you have more changes I'll wait to take a look.

I have to add the image name to Syringe priviliged list...  but it should be ok.  I'll add that now.

**Comment by cloudtoad** [comment]
> Is this ready to go from your perspective? I should have time later in the week to check this out and merge but if you have more changes I'll wait to take a look.

I forked/created a branch in Syringe and updated "config.go" to include a cvx with a specific version number: "antidotelabs/cvx-3.7.6" .    It should be good to go now, all of it.  It's PTR only, obviously.

File: images/cvx-3.7.6/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+FROM debian:stable
```
Comment (Mierdin) [inline images/cvx-3.7.6/Dockerfile]: I don't think this file is necessary - looks to be a duplicate of `images/cvx/Dockerfile`

File: images/cvx-3.7.6/Makefile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# SHELL=/bin/bash
```
Comment (Mierdin) [inline images/cvx-3.7.6/Makefile]: I don't think this file is necessary - looks to be a duplicate of `images/cvx/Makefile`

File: images/cvx-3.7.6/launch.sh [hunk 3]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```
Comment (Mierdin) [inline images/cvx-3.7.6/launch.sh]: I don't think this file is necessary - looks to be a duplicate of `images/cvx/launch.sh`

File: images/cvx/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    screen \
+    qemu-kvm \
+    ssh \
+    tcpdump \
+    ethtool \
+    telnet \
+    procps \
+ && rm -rf /var/lib/apt/lists/*
+
+COPY cvx-3.7.6.qcow2 /cvxi-3.7.6.qcow2
```
Comment (Mierdin) [inline images/cvx/Dockerfile]: Typo?

```suggestion
COPY cvx-3.7.6.qcow2 /cvx-3.7.6.qcow2
```

File: images/cvx/Makefile [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# SHELL=/bin/bash
+
+TARGET_VERSION ?= latest
+
+all: docker
+
+docker:
+	docker build -t antidotelabs/cvx-3.7.6:$(TARGET_VERSION) .
```
Comment (Mierdin) [inline images/cvx/Makefile]: We'll already be tagging the image with the curriculum version so let's omit the Cumulus version from the name to match the existing convention.

```suggestion
	docker build -t antidotelabs/cvx:$(TARGET_VERSION) .
```

File: images/cvx/Makefile [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+	docker push antidotelabs/cvx-3.7.6:$(TARGET_VERSION)
```
Comment (Mierdin) [inline images/cvx/Makefile]: Same as above.

```suggestion
	docker push antidotelabs/cvx:$(TARGET_VERSION)
```

File: images/cvx-3.7.6/Dockerfile (hunk 1 above)
Comment (cloudtoad) [reply images/cvx-3.7.6/Dockerfile]: Removed.

File: images/cvx-3.7.6/Makefile (hunk 2 above)
Comment (cloudtoad) [reply images/cvx-3.7.6/Makefile]: Removed

File: images/cvx-3.7.6/launch.sh (hunk 3 above)
Comment (cloudtoad) [reply images/cvx-3.7.6/launch.sh]: Removed

File: images/cvx/Dockerfile (hunk 4 above)
Comment (cloudtoad) [reply images/cvx/Dockerfile]: Fixed

File: images/cvx/Makefile (hunk 5 above)
Comment (cloudtoad) [reply images/cvx/Makefile]: Omitted.

File: images/cvx/Makefile (hunk 6 above)
Comment (cloudtoad) [reply images/cvx/Makefile]: Omitted

**Comment by cloudtoad** [comment]
Removed unnecessary directory.  Removed references to "3.7.6" in all files.  Added "gsutil cp" command to Makefile to download cvx image from gcp bucket.

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,9 +2,11 @@
 
 ## In development
 
+- Added Cumulus PTR demo [#253](https://github.com/nre-learning/nrelabs-curriculum/pull/253)
 
 ## v1.0.0 - August 08, 2019
 
+
```
Comment (Mierdin) [inline CHANGELOG.md]:
```suggestion
```

**Pull request merged**